### ISSUE TITLE
Add LICENSE

### ISSUE THREAD

**Issue by ahmetb** [matryer/gopherize.me#9]

**Comment by posener** [comment]
Can this project be cloned and changed to be used for other purposes?
Thanks

**Comment by bgadrian** [comment]
I this issue was fixed with  https://github.com/matryer/gopherize.me/commit/9a0c7024f7992d1af8ff3a69324acccf65bc538b

**Comment by markuswustenberg** [comment]
@matryer perhaps it would be nice to state on the website directly what license the generated artwork is provided under. Would you accept a PR for this?

**Comment by matryer** [comment]
@markuswustenberg that's not a bad idea, @ashleymcnamara what do you think about this?

**Comment by markuswustenberg** [comment]
@matryer any news on this? 😊

**Comment by matryer** [comment]
The artwork is licensed under the Creative Commons 3.0 Attributions license. Renee French is the original creator, although Ashley created the Gopherize.me style.

You can use them for open source projects, profile pics, presentations, etc. But avoid commercial stuff.

More info: https://golang.org/doc/gopher/README

I hope this helps.

**Comment by markuswustenberg** [comment]
Right, thanks. I was actually planning to use it for a commercial Go course, but I'll have to find something else. 😊

(I think this issue is still valid, it would probably be a good idea to put the license on the website.)
